src/widget/OktaSignIn.js

```diff
@@ -22,6 +22,10 @@ var OktaSignIn = (function () {
           _.extend({}, widgetOptions, renderOptions, {
             authClient: authClient,
             globalSuccessFn: (res) => {
+              const statusesToIgnore = ['UNLOCK_ACCOUNT_EMAIL_SENT', 'FORGOT_PASSWORD_EMAIL_SENT'];
```

Comment:
The early return logic should only affect the returned promise. The success callback should still fire as it has before.

Reply:
Done